src/site/_mixins.scss

```diff
@@ -183,6 +175,15 @@
 @mixin input {
   @include container;
 
+  &__label-container {
+    display: flex;
+    justify-content: space-between;
+    &>button[class*=__questionmark] {
+      position: unset;
+      right: unset;
```

Comment:
replace with inset-inline

Reply:
Done. Have shifted this code to description mixin. As it should apply wherever info icon is there, and some elements which don't use input mixin were left.